utils/prune_binaries.py

```diff
@@ -128,9 +129,12 @@ def _callback(args):
     prune_list = tuple(filter(len, args.pruning_list.read_text(encoding=ENCODING).splitlines()))
     unremovable_files = prune_files(args.directory, prune_list)
     if unremovable_files:
-        get_logger().error('%d files could not be pruned.', len(unremovable_files))
-        get_logger().debug('Files could not be pruned:\n%s',
-                           '\n'.join(f for f in unremovable_files))
+        file_list = 'n'.join(f for f in itertools.islice(unremovable_files, 5))
```

Comment:
The first newline is missing the backslash:  
```suggestion
        file_list = '\n'.join(f for f in itertools.islice(unremovable_files, 5))
```
<br>
Everything else looks good to me and works as expected.

Reply:
thanks, fixing this immediately